### ISSUE TITLE
how can i  cancel  the color that  the router-link  added ?

### ISSUE THREAD

**Issue by youngchen1987** [vuematerial/vue-material#653]
why the router-link wil add the color?

![image](https://cloud.githubusercontent.com/assets/11566474/24540367/7bc110de-1625-11e7-802a-d61a91ecf4e0.png)

and how can i cancel it, like primary? 
![image](https://cloud.githubusercontent.com/assets/11566474/24540450/de3db730-1625-11e7-9aba-8771f595873d.png)

![image](https://cloud.githubusercontent.com/assets/11566474/24540347/60b1306c-1625-11e7-8a86-071ba6ea6a35.png)

**Comment by Morgul** [comment]
You can change this by overriding the `linkActive` class:

http://router.vuejs.org/en/api/options.html#linkactiveclass

**Comment by alexvenetis** [comment]
Hello,

I'm also trying to customize the style of active routes, but can't seem to get anything to work.

I tried following the vue documentation, but I can't understand it. Not too helpful.
http://router.vuejs.org/en/api/options.html#linkactiveclass

I also tried following the vue-material documentation below, but it also doesn't work?
https://vuematerial.io/with-router

Does anyone have example of how to cutomize the style of vue-material active routes?
